### DISCUSSION TITLE
Location for adding physics tendency use case?

### DISCUSSION THREAD

**Discussion by bikegeek** [dtcenter/METplus#1858] [Use Cases] [answered]
I am in the process of adding a use case for creating the plots from David Ahijevych's FV3 physics tendency work.  I am unsure where to put the code.  Here are the available locations:

- air_quality_and_comp			
- climate					
- data_assimilation		
- marine_and_cryosphere			
- medium_range
- precipitation
- s2s	
- s2s_mjo
- s2s_mid_lat
- short_range
- space_weather
- tc_and_extra_tc

Or does this warrant a new category?

**Comment by JohnHalleyGotway** [comment]
@j-opatz and @georgemccabe can you two please comment on this discussion from @bikegeek? How should this new use case be categorized?

**Comment by TaraJensen** [comment, marked as answer]
@JohnHalleyGotway - being neither of them know much about it - I'll step in - I'd suggest short-range being that is where the work stems from, but it would be good to add a keyword for medium-range also.

**Reply by georgemccabe** [reply]
Thanks @TaraJensen. So use **short_range** as the use case category, but be sure to also add the **MediumRangeAppUseCase** quick search keyword in the use case documentation file so that it is searchable for medium range use cases.
